draft-ietf-quic-http.md

```diff
@@ -793,10 +793,15 @@ data from the target of the CONNECT.
 A TCP connection error is signaled by abruptly terminating the stream. A proxy
 treats any error in the TCP connection, which includes receiving a TCP segment
 with the RST bit set, as a stream error of type H3_CONNECT_ERROR; see
-{{errors}}.  Correspondingly, if a proxy detects an error with the stream or the
-QUIC connection, it MUST close the TCP connection.  If the underlying TCP
-implementation permits it, the proxy SHOULD send a TCP segment with the RST bit
-set.
+{{errors}}.
+
+Correspondingly, if a proxy detects an error with the stream or the QUIC
+connection, it MUST close the TCP connection.  Similarly, if the proxy receives
+a QUIC RESET_STREAM frame or STOP_SENDING frame for the stream, it MUST close
```

Comment:
It makes sense for RESET_STREAM to RST the TCP connection, but STOP_SENDING could be safely ignored because there's no equivalent in TCP?